### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by KrishanBhasin**

### CODE REVIEW

**Comment by KrishanBhasin** [comment]
Maybe WhatsApp should move above phone calls?

File: index.md [hunk 1]
```diff
@@ -178,9 +178,17 @@ Face to face communication can seem secure since there's nothing recording you.
 (You may think this is weird but I've recently noticed one of our local party executive members has been recording all our meetings with his phone, so that he can take notes later. He doesn't tell anyone he's doing that, nor do we know how carefully he handles/disposes of those recordings)
 
 ## Phone calls
-Phone calls are perfect for remotely sharing information without it being recorded (usually).
-- Before speaking to someone, ask them if they're on loudspeaker or in earshot of anyone else
-- Some people will use call recording applications on their phones, so assume they are
+Phone calls are perfect for remotely sharing information without it being recorded (usually). When making phone calls you have two options:
+
+1) **Use [WhatsApp](https://www.whatsapp.com/)** if you are trying to call someone on a mobile phone number, _and_ they have WhatsApp installed,
+    - It uses stronger protections than "normal" phone calls, so prefer this option wherever possible
```
Comment (JamesBelchamber) [inline index.md]:
```suggestion
    - Normal phone calls can be intercepted by criminals, calls through WhatsApp are encrypted
```

Comment (JamesBelchamber) [reply index.md]: Let's just clarify what "stronger" means. Otherwise all okay to be merged.

**Comment by KrishanBhasin** [comment]
Looks like this broke the markdown rendering on the actual site

**Comment by JamesBelchamber** [comment]
🤦‍♂ 

I'll fix this tomorrow, unless you want to 😅 in the medium term I suspect we'll want to port this to something like [Just the Docs](https://pmarsceill.github.io/just-the-docs/) or a similar documentation solution. May be worth doing that sooner rather than later.

**Comment by KrishanBhasin** [comment]
https://github.community/t5/GitHub-Pages/md-to-html-conversion-does-not-seem-to-work-properly/m-p/11289#M887

Something to do with the different rendering engines used...

**Comment by KrishanBhasin** [comment]
If you have a fix in mind go for it! I'm busy googling differences between GH markdown and "Kramdown" markdown...

**Comment by KrishanBhasin** [comment]
From https://kramdown.gettalong.org/syntax.html#lists:

>The original Markdown syntax allows the markers of ordered and unordered lists to be mixed, the first marker specifying the list type (ordered or unordered). This is not allowed in kramdown. As stated, the above example will give you two lists (an unordered and an ordered) in kramdown and only one unordered list in Markdown.

**Pull request merged**